assets/src/edit-story/app/media/useMediaReducer/reducer.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/* eslint complexity: ["error", { "max": 30 }] */
```

Comment:
Why is this needed? I don't think we should go over 21 if we can.

Reply:
There no complexity additions, but since this file gets size, it also increased the complexity of the  ESLint report. In general, use `switch` statements for reducers to create this type of report, so use a functional approach can be better. On this discussion https://github.com/google/web-stories-wp/pull/575#discussion_r395521374, we decided to improve it in another PR.

Reply:
@obetomuniz Can you check with @jauyong to create an issue for that follow-up refactoring improvement? Splitting reducers into separate files should be an XS / S task I think.